app/main/views/services.py

```diff
@@ -500,10 +500,15 @@ def update_section_submission(framework_slug, lot_slug, service_id, section_id,
                                 service_id=service_id,
                                 section_id=next_section))
     else:
-        return redirect(url_for(".view_service_submission",
-                                framework_slug=framework['slug'],
-                                lot_slug=draft['lot'],
-                                service_id=service_id))
+        return redirect("{}#{}".format(
+            url_for(
```

Comment:
`url_for` has a special keyword argument `_anchor` for URL anchors, so we can avoid a manual `.format` (http://flask.pocoo.org/docs/0.10/api/#flask.url_for)